pages/index.md

```diff
@@ -1,13 +1,36 @@
 ---
+layout: homepage
```

Comment:
To treat this page as HTML you also need to change the extension of the file to ".HTML" as well.